test/WebJobs.Script.Tests/EndToEndTestsBase.cs

```diff
@@ -44,5 +48,87 @@ public EndToEndTestsBase(TTestFixture fixture)
             Assert.True(trace.Contains(TestHelpers.RemoveByteOrderMarkAndWhitespace("script processed queue message")));
             Assert.True(trace.Contains(TestHelpers.RemoveByteOrderMarkAndWhitespace(messageContent)));
         }
+
+        protected async Task EasyTablesTest(bool writeToQueue = true)
```

Comment:
I didn't get around to testing all the platforms -- my machine isn't set up properly either. I'll come back to this after this is in.

---
In reply to: [56192837](https://github.com/Azure/azure-webjobs-sdk-script/pull/98#discussion_r56192837) [](ancestors = 56192837,56192241)